### ISSUE TITLE
Wrong hover on machine-scoped settings

### ISSUE THREAD

**Issue by aeschli** [microsoft/vscode#79977]
Verifying https://github.com/microsoft/vscode/issues/73985#issuecomment-522013185

`php.validate.executablePath` has the `machine`scope.

On a remote window, open the  workspace settings and paste
`"php.validate.executablePath": "asdad"`

Hover should say that it can only be applied in 'machine' settings.

![image](https://user-images.githubusercontent.com/6461412/63845780-d7850680-c98a-11e9-9994-1fcb86e25a8f.png)

**Comment by aeschli** [comment]
In a remote window, the setting can also be configured in a user setting.
https://github.com/microsoft/vscode/issues/73985#issuecomment-522013185 says it should only be allowed in machine settings, so not sure what's the expected behaviour.

**Comment by sandy081** [comment]
@aeschli 

> In a remote window, the setting can also be configured in a user settings (by adding it to the settings.json editor, no error hover)

This is correct. Updated the comment. 
A machine setting can be configured only at machine level (local or remote).

**Comment by sandy081** [comment]
> Hover should say that it can only be applied in 'machine' settings.

Since we do not surface machine settings in UI, I would rephrase as follows

`This setting can only be applied in user settings in local window or in remote settings in remote window.`
